[2015-11-13 09:31:05] request.INFO: Matched route "task". {"route_parameters":{"_controller":"TaskListBundle\\Controller\\TaskController::indexAction","_route":"task"},"request_uri":"http://localhost/task_manager/web/task/"} []
[2015-11-13 09:31:05] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-11-13 09:31:05] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.due AS due_4, t0.completed AS completed_5, t0.deleted AS deleted_6, t0.date_completed AS date_completed_7, t0.date_deleted AS date_deleted_8, t0.task_list_id AS task_list_id_9 FROM task t0':  SQLSTATE[42S02]: Base table or view not found: 1932 Table 'task_manager.task' doesn't exist in engine" at C:\xampp\htdocs\task_manager\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.due AS due_4, t0.completed AS completed_5, t0.deleted AS deleted_6, t0.date_completed AS date_completed_7, t0.date_deleted AS date_deleted_8, t0.task_list_id AS task_list_id_9 FROM task t0':\n\nSQLSTATE[42S02]: Base table or view not found: 1932 Table 'task_manager.task' doesn't exist in engine at C:\\xampp\\htdocs\\task_manager\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1932 Table 'task_manager.task' doesn't exist in engine at C:\\xampp\\htdocs\\task_manager\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:106, PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1932 Table 'task_manager.task' doesn't exist in engine at C:\\xampp\\htdocs\\task_manager\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:104)"} []
[2015-11-13 09:31:22] request.INFO: Matched route "task". {"route_parameters":{"_controller":"TaskListBundle\\Controller\\TaskController::indexAction","_route":"task"},"request_uri":"http://localhost/task_manager/web/task/"} []
[2015-11-13 09:31:22] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-11-13 09:31:22] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.due AS due_4, t0.completed AS completed_5, t0.deleted AS deleted_6, t0.date_completed AS date_completed_7, t0.date_deleted AS date_deleted_8, t0.task_list_id AS task_list_id_9 FROM task t0':  SQLSTATE[42S02]: Base table or view not found: 1932 Table 'task_manager.task' doesn't exist in engine" at C:\xampp\htdocs\task_manager\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.due AS due_4, t0.completed AS completed_5, t0.deleted AS deleted_6, t0.date_completed AS date_completed_7, t0.date_deleted AS date_deleted_8, t0.task_list_id AS task_list_id_9 FROM task t0':\n\nSQLSTATE[42S02]: Base table or view not found: 1932 Table 'task_manager.task' doesn't exist in engine at C:\\xampp\\htdocs\\task_manager\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1932 Table 'task_manager.task' doesn't exist in engine at C:\\xampp\\htdocs\\task_manager\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:106, PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1932 Table 'task_manager.task' doesn't exist in engine at C:\\xampp\\htdocs\\task_manager\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:104)"} []
[2015-11-13 14:54:51] request.INFO: Matched route "homepage". {"route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost/task_manager/web/"} []
[2015-11-13 14:54:51] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-11-13 14:54:51] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "default/index1.html.twig"." at C:\xampp\htdocs\task_manager\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"default/index1.html.twig\". at C:\\xampp\\htdocs\\task_manager\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"default/index1.html.twig\". at C:\\xampp\\htdocs\\task_manager\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Loader\\FilesystemLoader.php:91, InvalidArgumentException(code: 0): Unable to find template \"default/index1.html.twig\" : \"The file \"default/index1.html.twig\" does not exist (in: C:\\xampp\\htdocs\\task_manager\\app/Resources).\". at C:\\xampp\\htdocs\\task_manager\\app\\cache\\prod\\classes.php:809, InvalidArgumentException(code: 0): The file \"default/index1.html.twig\" does not exist (in: C:\\xampp\\htdocs\\task_manager\\app/Resources). at C:\\xampp\\htdocs\\task_manager\\app\\cache\\prod\\classes.php:1684)"} []
